packages/lib-classifier/.storybook/main.js

```diff
@@ -10,7 +10,7 @@ const config = {
     options: {}
   },
   docs: {
-    autodocs: 'tag'
```

Comment:
I'd like to keep the overall classifier setting as `'tag'` for now. Definitely feel free to enable docs in specific json subject viewer components though!

If we enable autodocs on all storybook components, the Docs page looks exactly like the Default story for the majority of components in the classifier, making it rather redundant than helpful. I'm definitely open to having a discussion about standardization of docs elements in the classifier though so that autodocs could eventually be enabled for all.

Reply:
Yeah, I put this into its own commit so that it would be easy to remove/revert. 

I’m finding the missing Docs pages, in the new storybook, really annoying, because I used them in Storybook 6.

Reply:
For example, the component docs I wrote for Engaging Crowds have all been removed, without discussion.